word2gauss/embeddings.pyx

```diff
@@ -249,6 +249,64 @@ cdef class GaussianEmbedding:
         self.acc_grad_sigma = _acc_grad_sigma
         self.acc_grad_sigma_ptr = &_acc_grad_sigma[0]
 
+    def update(self, N, init_params={
+            'mu0': 0.1,
+            'sigma_mean0': 0.5,
+            'sigma_std0': 1.0
+        }):
+        cdef np.ndarray[DTYPE_t, ndim=2, mode='c'] _mu
+        cdef np.ndarray[DTYPE_t, ndim=2, mode='c'] _sigma
+        cdef np.ndarray[DTYPE_t, ndim=1, mode='c'] _acc_grad_mu
+        cdef np.ndarray[DTYPE_t, ndim=1, mode='c'] _acc_grad_sigma
+        LOGGER.info("New vocab size %i" % N)
+        LOGGER.info("Old vocab size %i" % self.mu.shape[0])
+        if N > self.mu.shape[0]:
+
+            # there are new words in the vocabulary
+            # add more rows
+            n_words = N - self.mu.shape[0]
+            LOGGER.info("%i new words" % n_words)
+
+            # update mu with n_words more rows initialized randomly
+            _mu = np.vstack([self.mu, init_params['mu0'] * np.ascontiguousarray(
+                np.random.randn(n_words, self.mu.shape[1]).astype(DTYPE))])
+            self.mu = _mu
+            LOGGER.info("mu updated with %i new rows" % n_words)
+
+            # update sigma
+            s = np.random.randn(n_words, 1).astype(DTYPE)
```

Comment:
What if `self.covariance_type == DIAGNONAL`?  At least add an if statement here to make it easier to add support for this case in the future.